scripts/populate_mock_db.py

```
@@ -0,0 +1,75 @@
+    # Ensure directory exists
+    os.makedirs(os.path.dirname(DB_PATH), exist_ok=True)
+
+    if os.path.exists(DB_PATH):
+        os.remove(DB_PATH)
+
+    conn = init_db(DB_PATH)
+    c = conn.cursor()
+
+    # Add 50 crew members
+    roles = ["WC", "CM", "CC", "FFC", "FFD", "FFT"]
+    skills_pool = ["TTR", "LGV", "BA", "IC", "ERD"]
```

Ruff (0.15.0)

[error] Local variable `skills_pool` is assigned to but never used

Remove assignment to unused variable `skills_pool`

(F841)